test/myFTClient.spec.js

```diff
@@ -835,172 +835,197 @@ describe('myFT Client proxy', () => {
+		afterEach(done => {
+				nock.cleanAll();
+				done();
+		});
 
-			it('Should remove concept(s) followed by a single user', done => {
-				const userId = '00000000-0000-0000-0000-000000000002';
+		it('Should set concept(s) follows by a user ', done => {
 				nock(baseUrl)
-					.delete('/kat/user/follows',{
-						ids: [userId], // The next-myft-api expects an array of user IDs
-						subjects: subjects
-					})
+					.post('/kat/user/follows')
 					.query(true)
-					.reply(204, () => ({}));
-
-				myFT.removeConceptsFollowedByKatUser(userId, subjects)
-					.then(addResp => {
-						expect(addResp).to.be.an('Object');
-						expect(addResp.status).to.equal(204);
+					.reply(200, () => followedByKatRes);
 
-						done();
-					})
-					.catch(done);
-			});
+			myFT.addConceptsFollowedByKatUser(ids, subjects, relProps)
+				.then(addResp => {
+					expect(addResp).to.be.an('array');
+					expect(addResp[0][0][0]).to.have.deep.property('katRel.type', 'followed_by_kat');
+					done();
+				})
+				.catch(done);
+		});
 
-			it('Should remove concept(s) followed by a group', done => {
-					nock(baseUrl)
-						.delete('/kat/group/follows')
-						.query(true)
-						.reply(204);
+		it('Should get an array of concepts followed by a group', done => {
+				nock(baseUrl)
+					.get(`/group/${uuids.validLicence}/followed_by_kat/concept?page=1&limit=500`)
+					.reply(200, () => require('./mocks/fixtures/groupFollowedConcept'));
 
 
-				myFT.removeConceptsFollowedByKatGroup(ids, subjects)
-					.then(addResp => {
-						expect(addResp).to.be.an('Object');
-						expect(addResp.status).to.equal(204);
+			myFT.getConceptsFollowedByKatGroup(uuids.validLicence)
+				.then(followResponse => {
+					expect(followResponse).to.be.an('array');
+					expect(followResponse).to.have.lengthOf(1);
+					expectOwnProperties(followResponse, ['uuid']);
 
-						done();
-					})
-					.catch(done);
-			});
+					done();
+				})
+				.catch(done);
+		});
 
-			it('Should add concept(s) follows for members of a group', done => {
+		it('Should set concept(s) follows by a group ', done => {
 
-					nock(baseUrl)
-						.post('/kat/group/user/follows')
-						.query(true)
-						.reply(200, () => []);
+				nock(baseUrl)
+					.post('/kat/group/follows')
+					.query(true)
+					.reply(200, () => followedByKatRes);
 
+			myFT.addConceptsFollowedByKatGroup(ids, subjects, relProps)
+				.then(addResp => {
+					expect(addResp).to.be.an('array');
+					expect(addResp[0][0][0]).to.be.an('Object');
+					expect(addResp[0][0][0]).to.have.deep.property('katRel.type', 'followed_by_kat');
+					//TODO specify what to expect from addResp
+					done();
+				})
+				.catch(done);
+		});
 
-				myFT.addConceptsFollowedByKatGroupMembers(ids, subjects)
-					.then(addResp => {
-						expect(addResp).to.be.an('array');
-						//TODO specify what to expect from addResp Postman returns an empty array in an empty array?
+		it('Should remove concept(s) followed by a single user', done => {
```

Comment:
For this and the other tests after it, I'm not sure what we are testing matches the title of the test, and it feels like we are just testing what has been returned by `nock`.
If the end goal of the function is to call a `next-myft-api` endpoint with specific data, maybe we should be testing for that.